package.json

```diff
@@ -19,18 +19,21 @@
   },
   "scripts": {
     "pretty": "prettier --single-quote --trailing-comma es5 --write {middleware,routes,shopStore}{/*,/**/*}.js",
-    "precommit": "yarn run pretty"
+    "precommit": "yarn run pretty",
```

Comment:
This is unrelated but I'm pretty sure this will never work without [Husky](https://github.com/typicode/husky). Issue opened: https://github.com/Shopify/shopify-express/issues/22

Reply:
right good catch